modules/custom/openy_addthis/openy_addthis.install

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @file
+ * Installation file for OpenY addThis module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function openy_addthis_install() {
```

Comment:
I'm sorry, I've mixed up modules.
This function should be in demo module, `modules/openy_features/openy_demo_content/modules/openy_demo_addthis/openy_demo_addthis.install`.